app/components/atomic/galoy-icon-button/galoy-icon-button.tsx

```diff
@@ -53,7 +54,7 @@ export const GaloyIconButton = ({
         return {
           opacity: 0.7,
           color: colors.primary,
-          backgroundColor: colors.grey4,
+          backgroundColor: "transparent",
```

Comment:
we should have a colors.transparent instead of hardcoding it here

Reply:
Will make that change